MonkeyLoader/vendor/run_monkeyloader.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/sh
+# Doorstop start script
+# 
+# Run the script to start the game with Doorstop enabled
+#
+# There are two ways to use this script
+#
+# 1. Via CLI: Run ./run.sh <path to game> [doorstop arguments] [game arguments]
+# 2. Via config: edit the options below and run ./run.sh without any arguments
+
+# 0 is false, 1 is true
+
+# LINUX: name of Unity executable
+# MACOS: name of the .app directory
+executable_name="Resonite.x86_64"
```

Comment:
Technically this should probably be left blank, and then modified in from the resonite game pack repo's build... but then again, this vendoring should probs be done in the unity gamepack anyways...

Am not letting best be the enemy of better tho so shipping it like this for now is fine IMO since Resonite is realistically the only game so far where am aware that monkeyloader is being used....

Reply:
yeaaa, true that :'D

could you approximate it by getting the executable name from the folder name it's in? 🤔

Reply:
I mean, of course _could_, but not sure it's worth the effort, given that linux users generally are used to things being a bit jank anyways :stuck_out_tongue: and there can be many different folder setups and executables for different games, trying to correctly parse what should be launched just sounds like overly difficult vs just letting users modify this...

Reply:
True that.